lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.html

```diff
@@ -1,5 +1,6 @@
 <form [formGroup]="form" novalidate (ngSubmit)="apply(form.value, form.valid)">
     <mat-form-field>
+        <mat-label>{{ 'SEARCH.FILTER.RANGE.SELECT-FROM-DATE' | translate }}</mat-label>
```

Comment:
I think we should use floating label as described in material documentation https://material.angular.io/components/form-field/overview#floating-label to not disrupt the UI in the form. Otherwise please consult designer or PM if non-floating label is fine here.

Reply:
Btw, you didn't specify `for` attribute, without it `MatFormField` wouldn't know which label is associated with which field which will cause another accessibility issue in the future... Please read accessibility section in the documentation https://material.angular.io/components/form-field/overview#accessibility